performance_manager/load_vp_events.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import os
+import sys
+import pathlib
+
+from typing import List
+
+import sqlalchemy
+from sqlalchemy import insert
+from sqlalchemy import select
+from sqlalchemy import update
+from sqlalchemy.orm import sessionmaker
+
+from lib import (
+    SqlBase,
+    MetadataLog,
+    get_experimental_engine,
+    get_local_engine,
+    get_vp_dataframe,
+    transform_vp_dtyes,
+    transform_vp_timestamps,
+    merge_vehicle_position_events,
+)
+
+logging.getLogger().setLevel("INFO")
+DESCRIPTION = """Entry Point to RDS Manipulating Performance Manager Scripts"""
```

Comment:
Tried to model this after the existing `thing_to_run` script. Going forward I think we'll need a more standard way of encapsulating these job scripts to be run more uniformly.

Reply:
yeah. we def need something more. `thing_to_run` was just a script i was using for grasping around in the dark while learning how these tools worked.